src/store/record.js

```diff
@@ -163,6 +160,7 @@ let recordStore = {
             }
         },
         cleanRecordStore(state){
+            state.sections = null;
```

Comment:
here webstorm warns. I changed the value to undefined or {} and the error was gone. 
![image](https://user-images.githubusercontent.com/25683981/108864015-b3d76680-75e9-11eb-98cd-bcaa326d015b.png)